introduction.md

```diff
@@ -0,0 +1 @@
+Hi there. My name is Joao and I'm following the github course from code school.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.